roles/telemetry_autoscaling/tasks/test_autoscaling.yml

```diff
@@ -62,6 +62,7 @@
     # source ~/overcloudrc;
     {{ openstack_cmd }} alarm list -c state -c name -f value| \
         grep -i "cpu_alarm_high" | \
+        grep -i "{{ stack_name }}" | \
```

Comment:
I added a grep for the stack_name. This is just to allow to run these tests when there is already an autoscaling stack with a different name present in the cluster.